movai_core_shared/envvars.py

```diff
@@ -10,14 +10,27 @@
     STRESS_INFLUX_DB,
 )
 
+LOG_LEVELS = {
+    
+    "NOTSET": NOTSET,
+    "INFO": INFO,
+    "DEBUG": DEBUG,
+    "WARNING": WARNING,
+    "ERROR": ERROR,
+    "CRITICAL": CRITICAL,
+    "FATAL": FATAL,
+}
+
+
 # Setting for logging verbosity levels
 # Will be set only once at startup
-MOVAI_STDOUT_VERBOSITY_LEVEL = int(os.getenv("MOVAI_STDOUT_VERBOSITY_LEVEL", str(DEBUG)))
-MOVAI_FLEET_LOGS_VERBOSITY_LEVEL = int(os.getenv("MOVAI_FLEET_LOGS_VERBOSITY_LEVEL", str(DEBUG)))
+MOVAI_STDOUT_VERBOSITY_LEVEL = DEBUG # Minimum log supported by handlers
+MOVAI_FLEET_LOGS_VERBOSITY_LEVEL = int( LOG_LEVELS.get(os.getenv("MOVAI_FLEET_LOGS_VERBOSITY_LEVEL", "DEBUG").upper().strip(), DEBUG)) # Verbosity level for influxdb logs
```

Comment:
not sure to understand this complicated syntax, I see 2 things we have to support 
- retrocompatible passing of level as integers
- user friendly  passing of level as strings
+ handling of other strings
normally `getLevelName` is enough see example below:

```
import logging
import os

def get_log_level(env_var_name, default=logging.INFO):
    """ Get the log level from the environment variable `env_var` and return it as a logging level """
    level = os.getenv(env_var_name, default)  # Default to 'INFO' if not set

    try:
        # Try converting to an integer level
        level = int(level)
    except ValueError:
        # If it's not an integer, it must be a string level
        pass

    # handle failure of getLevelName
    ret_level = logging.getLevelName(level)
    if isinstance(ret_level, str) and ret_level.startswith('Level '):
        print('Invalid log level: %s' % ret_level)
        return default

    return ret_level

# Test
os.environ["TEST"]="WARNING"
print(get_log_level("TEST"))  # Output: 30 (WARNING)
logging.basicConfig(level=get_log_level("TEST"))

os.environ["TEST"]="10"
print(get_log_level("TEST"))  # Output: DEBUG
logging.basicConfig(level=get_log_level("TEST"))

os.environ["TEST"]="toto"
print(get_log_level("TEST"))  # Output: 20 (INFO)
logging.basicConfig(level=get_log_level("TEST"))
```